report/report.tex

```diff
@@ -186,6 +186,8 @@ \subsection{Data Preprocessing}
 
 
 \subsection{Model building}
+To begin our model building LinearSVC was chosen as the initial classifier for our dataset. Initially the model did not converge in time. We therefore tried different max iterations to see when the model would converge and avoid overfitting and find the best fit for classifying a review as helpful/unhelpful. Performance does take a hit as the data size increases, see \figurename{\ref{Si_Results}}.
```

Comment:
Måske nævne som noget af det første, at LinearSVC forsøger at classifie vores dataset i to grupper, help og unhelpful, siden vores problem er binært af natur :D

Reply:
Done